docs/semgrep-ci/manual-ci-setup.md

```diff
@@ -388,9 +410,9 @@ TODO
 
 ### Ignoring files
 
-By default Semgrep CI skips files and directories such as tests/, node_modules/, and vendor/. It uses the same default .semgrepignore as the CLI, which can be found in the [CLI Reference](https://semgrep.dev/docs/cli-reference/#ignoring-files). This is used by Semgrep CI when no explicit .semgrepignore file is found in the root of your project.
+By default `semgrep ci` skips files and directories such as `tests/`, `node_modules/`, and `vendor/`. It uses the same default `.semgrepignore` as the `semgrep` command. This default `semgrepignore`  can be found in the [CLI Reference](https://semgrep.dev/docs/cli-reference/#ignoring-files). This is used by `semgrep ci` when no explicit `.semgrepignore` file is found in the root of your repository.
 
-Use a `.semgrepignore` file to exclude files and directories from a scan. The `.semgrepignore` file follows `.gitignore` syntax. You can exclude vendored code and code for tests through this feature.
+You can copy and commit the default `.semgrepignore` and extend it with your own entries or write one from scratch. `.semgrepignore` follows `.gitignore` syntax. If Semgrep detects a `.semgrepignore` file within your repository, it won't append entries from the default `.semgrepignore` file.
```

Comment:
"If Semgrep detects a `.semgrepignore` file within your repository"
This makes it sound a little like you can place your own .semgrepignore literally anywhere in your repo? Is that true? These two paragraphs are confusing about placement of the default an your own custom .semgrepignore.

Reply:
I have added its location (at the root of the repository). Thank you @efortuna!